blog/2023-02-22-consensus.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Consensus Team Update
+slug: 2023-02-22-consensus
+authors: dnadales
+tags: [consensus]
+hide_table_of_contents: false
+---
+
+## High level summary
+
```

Comment:
This is still missing a high level summary? Or shall we just remove this section

Reply:
On Slack, by @dnadales:

> (I'll add the high-level summary after I meet with Arnaud)